lusSTR/tests/data/Flanks_testing_file.csv

```diff
@@ -0,0 +1,1095 @@
+Locus,Reads,ForenSeq,Sample,Analysis,Sample,Allele,Locus
```

Comment:
I'm just noticing this now, maybe it's been here all along, but there are two columns named "Sample" with different values?

Reply:
Ah, that's supposed to be the Project. I removed the last two columns too since they aren't necessary for this (I had used this file for something else).